prog/vm/github_runner.rb

```diff
@@ -138,13 +129,15 @@ def before_run
       github_runner.update(runner_id: response[:runner][:id], ready_at: Time.now)
       # ./env.sh sets some variables for runner to run properly
       vm.sshable.cmd("./env.sh")
-      vm.sshable.cmd("common/bin/daemonizer 'sudo -u runner /home/runner/run.sh --jitconfig #{response[:encoded_jit_config].shellescape}' runner-script")
+
+      command = "/bin/bash -c 'sudo -u runner ./run.sh --jitconfig #{response[:encoded_jit_config].shellescape}'"
```

Comment:
since you don't do input/output redirection, you don't need /bin/bash anymore.

Reply:
Removed it